resources/views/api/pages/comm_links/show.blade.php

```diff
@@ -60,23 +60,23 @@
                         @lang('Metadaten')
                     </a>
 
-                    <a class="nav-item nav-link" id="nav-api-tab" href="{{ route('api.v2.cl.show', $commLink->cig_id) }}" aria-selected="false" target="_blank">
+                    <a class="nav-item nav-link" id="nav-api-tab" href="{{ app('api.url')->version('v1')->route('api.v1.rsi.comm-links.show', $commLink->cig_id) }}" aria-selected="false" target="_blank">
```

Comment:
This seems to be a merge artifact. The `app('api.url')` stuff does not exist anymore